exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestMetastoreCommands.java

```diff
@@ -3531,6 +3531,40 @@ public void testAnalyzeWithNonWritableWorkspace() throws Exception {
     run("analyze table dfs.%s.%s refresh metadata", workspaceName, tableName);
   }
 
+  @Test
+  public void testAnalyzeAllTypes7kRows() throws Exception {
+    // See DRILL-7968.  More rows are produced here to test an ANALYZE code path
```

Comment:
May be shortened to : 
```java
@Test // DRILL-7968
```

Reply:
Thank you, I've applied the fixes and a build+test run should kick off presently.